scripts/README.md

```diff
@@ -0,0 +1,29 @@
+# Scripts for generating figures and tables
+
+This folder contains all scripts used to generate figures and tables.
+
+## Figure set up
+
+The `figure_setup` folder contains scripts that are required to be run _prior to_ generating figures.
+
+1. `sync-metadata.R`: This script is used to sync any metadata files found on S3 to a local folder.
+In particular, the `scpca-sample-metadata.tsv` and `scpca-library-metadata.tsv` will be stored to a folder within the root directory of this repo named `s3_files`.
+In order to generate some of the figures (see more on which figures require this script below), this script will be need to run first.
+To run the script use the following command:
+
+```sh
+op run -- Rscript sync-metadata.R
+```
+
+## Generating figures
+
+The following scripts can be used to generate figures:
+
+1. `fig_1a-sample_disease_barchart.R`: This script is used to generate Figure 1A, which includes a summary of the types of diagnoses found in the ScPCA portal.
+Before running this script, you must run `figure_setup/sync-metadata.R`.
+
+2. `fig_1b-modality_barchart.R`: This script is used to generate Figure 1B, which includes a summary of the types of modalities found in the ScPCA portal.
+Before running this script, you must run `figure_setup/sync-metadata.R`.
+
+3. `table_s1-modality_summary.R`: This script is used to generate supplemental Table 1, which contains a summary of the types of libraries found in each project.
```

Comment:
Place this in a new section `## Generating tables` as item 1, or change the header above to `## Generating figures and tables`. You choose the approach! Just keep it consistent with however you choose to organize in the root README.

Reading this list also bought up a question, do we want to wrap all these in a bash script, or two scripts? (figures & tables, or main & SI). Adding that would be a separate issue/PR!

Reply:
I was thinking that it would also make the GHA easier!